src/dlgConnectionProfiles.cpp

```diff
@@ -336,11 +336,12 @@ void dlgConnectionProfiles::slot_save_name()
 
         pItem->setText(newProfileName);
 
-        QDir currentPath(QStringLiteral("%1/.config/mudlet/profiles/%2").arg(QDir::homePath(), currentProfileEditName));
+        QDir currentPath(mudlet::getMudletPath(mudlet::profileHomePath, currentProfileEditName));
         QDir dir;
 
         if (currentPath.exists()) {
-            QDir parentpath(QStringLiteral("%1/.config/mudlet/profiles/").arg(QDir::homePath()));
+            // CHECKME: previous code specified a path ending in a '/'
```

Comment:
So should this be checked?

Reply:
I seek feedback as to whether the '/' is needed or not - I do not _think_ so but _I think rain is wet so who am I to say..._